.github/workflows/r.yml

```diff
@@ -89,7 +89,9 @@ jobs:
         run: |
           sudo sysctl -w kernel.core_pattern="core.%e.%p"
           ulimit -c unlimited
-          archery docker run ubuntu-r
+          # Setting a non-default and non-probable Marquesas French Polynesia time
+          # it has both with a .45 offset and very very few people who live there.
+          archery docker run -e TZ=MART ubuntu-r
```

Comment:
Are you *sure* this works? I thought `-e` was for passing env vars into the runtime environment in the docker container, but you want to parametrize the dockerfile.

Reply:
Yeah, you're right — this isn't totally parameterized the way we want. 

The outcome _is_ similar to what we want: `TZ` is set to `"UTC"` (really any valid value that's not `""` will get rid of the nuisance warning) and then when during the docker run when the tests are run `TZ` is set to `"MART"` and so the tests are run in a non-UTC timezones (which is when we care about it being an improbable timezone). 

It shouldn't matter what the TZ is set to during the build process, but I'm going to parameterize this fully so that this confusion doesn't bite us elsewhere.